config/licenses/unported.js

```diff
@@ -0,0 +1,688 @@
+/* eslint-disable prettier/prettier */
+
+// Array of licenses and their attributes:
+//
+//  * id
+//  * name
+//  * groups
+//  * compatible licences
+//  * regexp
+//  * url
+//
+// Licenses are sorted by restrictiveness and are matched in order:
+//
+// If an asset features multiple licenses, only the first one
+// featured in this list will be detected.
+//
+// Ported from: https://github.com/wmde/Lizenzhinweisgenerator/blob/0df1805ef1b362b56c287b7035b2bbf66e6642a1/js/app/LICENCES.js
+//
+module.exports = [
```

Comment:
@juffel had the idea to store them as CSV for better maintainability. Not sure how doable this is when having regular expressions in them... What do you think?

Reply:
🤔 You can certainly create `RegExp`s from strings in JavaScript [[1]](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp):

```javascript
new RegExp('^[a-z]+')
```

Not sure how much value it adds for now 🤷‍♂️

Reply:
Oh, I didn't mean necessarily to store them as CSV. I just wanted to propose to store them in a purely static fashion, without `new License(...)` statements in the array 😉 
I like this ☝️ solution a lot!